docs/release-notes.md

```diff
@@ -4,6 +4,43 @@ nav_order: 9
 
 # Release notes
 
+## Butane 0.15.0 (2022-06-23)
+
+### Breaking changes
+
+- Transition to explicit compression signature for MakeDataURL() _(Go API)_
+
+### Features
+
+- Add arm64 support
+- Add Flatcar support 1.0 and 1.1-exp, targeting Ignition spec 3.30 and 3.40-exp respectively
+- Stabilize OpenShift spec 4.11.0, targeting Ignition spec 3.2.0
+- Add GRUB password support _(openshift 4.12.0-exp)_
+- Extended TranslationSet; AddFromCommonObject() TranslationSet.Map(); _(Go API)_
+
+### Bug fixes
+
+- Fix 'build' script for Windows
+- Fix local file inclusion on Windows
+
+### Misc. changes
+
+- Improve cross-platform consistency 
+- Derive container from Fedora for multi-stage build support
+- Fail if setuid/setgid/sticky mode bits specified
+- Update to ignition v2.14.0
+- Roll back to Ignition spec 3.2.0 _(openshift 4.11.0)_
+
+### Docs changes
+
+- Update docs/examples.md to use containerized mkpasswd
+- Clarify setuid/setgid/sticky mode bits support
```

Comment:
We're not really clarifying, since we didn't document it at all before.  And it's different in each spec.  Maybe something like:

```suggestion
- Document support status of setuid/setgid/sticky mode bits in each spec
```